css/buttons.css

```diff
@@ -4,11 +4,12 @@
   border: 0.5px solid var(--slate-300);
   width: max-content;
   height: max-content;
-  padding: 0.7rem 1rem;
+  padding: 0.7em 1em;
```

Comment:
would suggest using rem here

Reply:
you've replaced it with `em`  but i think you meant to make it `rem`